src/init/init.ts

```diff
@@ -156,12 +156,11 @@ function getDisplayName(generatorName: string) {
   // -------------------------------------------------------------------
   // (generator-)?           | Grp 1; Match "generator-"; Optional
   // (polymer-init)?         | Grp 2; Match "polymer-init-"; Optional
-  // (polymer-starter-kit-)? | Grp 3; Match "polymer-starter-kit-"; Optional
```

Comment:
Other cleanup changes that should be done:
 1. Remove `"polymer-starter-kit-"` from function comment at line [140](https://github.com/Polymer/polymer-cli/pull/528/files#diff-6eb970fedd99f33facb11eefde800f22L140).
 2. Update `group 4 in pattern` to `group 3 in pattern` in comment at line [153](https://github.com/Polymer/polymer-cli/pull/528/files#diff-6eb970fedd99f33facb11eefde800f22L153).